addons/udes_stock/models/product_template.py

```diff
@@ -78,6 +87,25 @@ def _constrain_tracking_in_allowed_tracking_values(self):
                         % tracking_types_mapping[product_template.tracking]
                     )
 
+    def _compute_multiple_barcodes_config(self):
+        """Computed field to show the configuration set on the warehouse for multiple barcodes."""
+        warehouse = self.env.ref("stock.warehouse0")
+        for product in self:
+            product.u_multiple_barcodes = warehouse.u_product_multiple_barcodes
+
+    @api.depends("product_variant_ids", "product_variant_ids.u_barcode_ids")
+    def _compute_barcode_ids(self):
+        for p in self:
+            if len(p.product_variant_ids) == 1:
```

Comment:
Q: If there is more than 1 variant, do we set u_barcode_ids on the variant instead?